proposals/75:upgrade-16/use.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
empty. consider dropping to aid debugging in the future (clear there's no "use" stage)